README.md

```diff
@@ -145,7 +145,8 @@ The binary will be found at `target/release/sheldon`.
 ### Initializing
 
 Sheldon works by specifying plugin information in a [TOML](https://toml.io)
-configuration file. You can initialize this file by running `sheldon init`.
+configuration file, `plugins.toml`. You can initialize this file by running
```

Comment:
This `README.md` file is automatically generated from the files in the `docs/` directory, so could you make these changes in `docs/src/Getting-started.md` and then run the following to auto-update the README.

```sh
cargo run --package generate-readme
```

Reply:
Ah, apologies; I had missed that. It will be fixed in the amended commit.